fe/fe-core/src/main/java/com/starrocks/backup/Repository.java

```diff
@@ -169,6 +169,15 @@ public long getCreateTime() {
 
     // create repository dir and repo info file
     public Status initRepository() {
+        Status st = initRepositoryInternal("__palo_repository_");
+        if (st.ok()) {
+            return Status.OK;
+        }
+        return initRepositoryInternal("__starrocks_repository_");
+    }
+
+    public Status initRepositoryInternal(String path) {
+        PREFIX_REPO = path;
```

Comment:
PREFIX_REPO is a static property, changing it may cause concurrent problem, because all the instance of Repository class share it.

Reply:
I have change PREFIX_REPO to non-static value. Because the repo maybe in different remote path. Now, different remote path may have different PREFIX_REPO.